packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -50,6 +52,21 @@ export const ProposalPreview = () => {
   const votingRounds = useVotingRounds(proposal?.votes, proposal?.proposalStatusUpdates)
   const [currentVotingRound, setVotingRound] = useState(0)
   const votes = votingRounds[currentVotingRound] ?? votingRounds[0]
+  const notVoted = useMemo(() => {
+    if (!proposal || !isActiveProposalStatus(proposal.status)) {
+      return
+    }
+
+    const votedMembers = Array.from(votes.map.values())
+      .flat()
+      .map((vote) => vote.voter)
+    const councilMembers = council?.councilors.map((councilor) => councilor.member)
+    return (
+      councilMembers?.filter(
+        (member) => !votedMembers.some((voted) => voted.id === member.id) && member.id !== proposal.proposer.id
```

Comment:
I think the proposers should appear in "not voted", since they can also vote on their own proposals. e.g https://dao.joystream.org/#/proposals/preview/184